students/10/main.js

```diff
@@ -34,6 +35,20 @@ const ex6 = async () => {
 // Your functions here...
 //
 
+//ex2
+const countCharsInBody = async (userId) => {
+    let count = 0;
+    try {
+    let result = await axios.get(`https://jsonplaceholder.typicode.com/posts/${userId}`);
```

Comment:
We have to watch our indentation.  Can you please make this look nice?  This will matter when we start using Python.

Reply:
I meant to go back in and format it with prettier